.env.example

```diff
@@ -1,7 +1,8 @@
+RN_TEXTILE_LB_URL="optional-lb-url"
+RN_TEXTILE_CAFE_URL="optional-cafe-url"
 RN_TEXTILE_CAFE_TOKEN="a token"
-RN_TEXTILE_CAFE_OVERRIDE="optional-cafe-url"
-RN_TEXTILE_CAFE_GATEWAY_URL="https://cafe.textile.io"
 RN_TEXTILE_CAFE_API_PIN_PATH="/a/pin/path"
+RN_TEXTILE_GATEWAY_URL="https://cafe.textile.io"
```

Comment:
Cleaning up the names. You either specify the load balancer api url, or a cafe url. And you supply a gateway url.